packages/react/src/app-layout/AppLayoutSidebarNav.tsx

```diff
@@ -254,8 +265,11 @@ function AppLayoutSidebarNavItemInner({
 					alignItems: 'center',
 
 					gap: mapSpacing(0.75),
-					color: boxPalette[isActive ? 'foregroundText' : 'foregroundAction'],
-					...(isActive && {
+					color:
+						boxPalette[
+							isActive || isCurrentPage ? 'foregroundText' : 'foregroundAction'
```

Comment:
So weird putting conditionals & ternaries inside an object lookup. I think it would be clearer just to do 2 `boxPalette[foos]` depending on the condition

Reply:
Yeah, sure. I have no strong opinions here.

- [ ] more boxPalette

Reply:
This still seems wrong though, because I don't know why the sub levels are black in the "whenActive" mode only 
<img width="283" alt="image" src="https://github.com/agriculturegovau/agds-next/assets/853552/b9af0342-368e-44ef-a00c-873d8a1beee1">

Reply:
It's less about the mode and more about the active group.

![image](https://github.com/agriculturegovau/agds-next/assets/43688949/23ae4ec8-2fd1-4f14-9e72-1adb0eb8d760)

Reply:
Ah. But again, I don't know where the actual final design is, but all the sub levels are blue unless active here: https://www.figma.com/design/VjfFBpZsSddcm7pFGcc9CU/Component-development-and-testing?node-id=12904-27248&t=XRQIdziEgCKnHBfr-0

Reply:
Or try this link: https://www.figma.com/design/SgSHfK8AUadp7aEzD34ZG3/AgDS---Agriculture-Design-System?node-id=28862-77780&m=dev

Reply:
So that suggests it should be blue by default 
<img width="674" alt="image" src="https://github.com/agriculturegovau/agds-next/assets/853552/cd6984d6-a52a-4740-80b3-dc4ffb7977f6">

Reply:
Oh wait, level two doesn't have an active group state, that was level 1. My bad, I'll update this.
- [ ] level two active group should only have left border not text colour change